src/main/resources/db/changeLog.xml

```diff
@@ -165,4 +165,14 @@
             <column name="modifiedat" type="datetime(6)"/>
         </createTable>
     </changeSet>
+
+    <changeSet id="taskparams" author="daniil smirnov">
+        <addColumn tableName="task">
+            <column name="assigneeids" type="JSON"/>
+            <column name="userrole" type="varchar(255)"/>
+            <column name="estimatedduration" type="varchar(255)"/>
+            <column name="estimatedreadiness" type="varchar(255)"/>
+            <column name="attachmenturls" type="JSON"/>
```

Comment:
Not sure how to do those JSON lists correctly, json is probably not even a good fit here